framework/src/timeintegrators/ExplicitTimeIntegrator.C

```diff
@@ -53,6 +53,9 @@ ExplicitTimeIntegrator::ExplicitTimeIntegrator(const InputParameters & parameter
 
   if (_solve_type == LUMPED || _solve_type == LUMP_PRECONDITIONED)
     _ones = addVector("ones", false, PARALLEL);
+
+  // ignore all common SNES-related petsc options to prevent unused option warnings
+  Moose::PetscSupport::ignoreCommonSNESOptions(_fe_problem);
```

Comment:
This change shows how the new `ignored_options` attribute in `PetscOptions` is utilized to disable options specific to the `ExplicitTimeIntegrator` that are not used.